packages/docs/scripts/bump-docs.js

```diff
@@ -1,85 +1,94 @@
-  const tempDir = tempDirObj.name
-   
+  const tmpDir = tmp.dirSync().name
+
   try {
-    const parentDir = path.resolve('..')
-    const outputDir = path.resolve('docs')
-    const websiteDir = path.resolve(outputDir, 'website')
-    const apiDir = path.resolve(websiteDir, 'build', 'api')
-    
-    //kernel
-    // const kernelRepoDir = path.resolve(tempDir, 'kernel')
-    // const kernelContractsDir = path.resolve(kernelRepoDir, 'contracts')
-    // shell.cd(tempDir)
-    // handleErrorCode(shell.exec('git clone https://github.com/zeppelinos/kernel.git'))
-    // shell.cd('kernel')
-    // //handleErrorCode(shell.exec(`git checkout -b ${tag} ${tag}`))
-    // handleErrorCode(shell.exec(`npx solidity-docgen ${kernelRepoDir} ${kernelContractsDir} ${outputDir}`))
-    
-    shell.cd(tempDir)
-    const libImportsDir = path.resolve('.', 'openzeppelin-solidity')
-    handleErrorCode(shell.exec('git clone https://github.com/openzeppelin/openzeppelin-solidity.git'))
-    shell.cd('openzeppelin-solidity')
-    handleErrorCode(shell.exec(`git checkout -b v1.8.0 v1.8.0`))
-    shell.cd('..')
-
-
-    //zos-lib
-    const libRepoDir = path.resolve('.', 'zos-lib')
-    const libContractsDir = path.resolve(libRepoDir, 'contracts')
-    //handleErrorCode(shell.exec('git clone https://github.com/zeppelinos/zos-lib.git'))
-    shell.cd('zos-lib')
-    //handleErrorCode(shell.exec(`git checkout -b ${tag} ${tag}`))
-    shell.cd('..')
-    // handleErrorCode(shell.exec(`npx solidity-docgen ${libRepoDir} ${libContractsDir} ${outputDir}`))
-    //handleErrorCode(shell.exec(`SOLC_ARGS=\'zeppelin-solidity=${libImportsDir}\' npx solidity-docgen ${libRepoDir} ${libContractsDir} ${outputDir}`))
-    
-
-    //zos-cli
-    const cliRepoDir = path.resolve('.', 'zos-cli')
-    const cliContractsDir = path.resolve(cliRepoDir, 'contracts')
-    // const cliImportsDir = path.resolve(libRepoDir, 'node_modules/zeppelin-solidity')
-    // handleErrorCode(shell.exec('git clone https://github.com/zeppelinos/zos-cli.git'))
-    // shell.cd('zos-cli')
-    // // handleErrorCode(shell.exec(`git checkout -b ${tag} ${tag}`))
-    // handleErrorCode(shell.exec(`npx solidity-docgen ${cliRepoDir} ${cliContractsDir} ${outputDir}`))
-    
-    shell.cd(websiteDir)
-    handleErrorCode(shell.exec(`yarn install`))
-    handleErrorCode(shell.exec(`npm run version ${version}`))
-    handleErrorCode(shell.exec('npm run build'))
-  }
-  finally {
-    shell.rm('-rf', tempDir)
+    const localDocsDir = path.resolve(__dirname, '../', 'docs')
+    const localBuiltDocsDir = path.resolve(localDocsDir, 'docs')
+    const localWebsiteDir = path.resolve(localDocsDir, 'website')
+    const localSidebarFile = path.resolve(localWebsiteDir, 'sidebars.json')
+    const packagesDir = path.resolve(tmpDir, 'zos', 'packages')
+    const builtDocs = path.resolve(packagesDir, 'docs', 'docs', 'docs')
+
+    cd(tmpDir)
+    exec('git clone https://github.com/zeppelinos/zos.git')
```

Comment:
I'd consider running `git clone` targeted at the local folder, so it's a local copy instead of fetching everything from github. Alternatively, it seems you can also use [`git worktree`](https://git-scm.com/docs/git-worktree) or somehow set `GIT_WORK_TREE` to checkout into a different location, without needing to copy the entire .git history.